src/electron/src/views/DataExportView.vue

```diff
@@ -219,8 +220,9 @@ function revealItemInFolder(event: Event) {
               <p>
                 Please click "Next" once you are ready to
                 <b class="dark:text-white">first review, and later share your data</b>. The export
-                that will be created with your permission in the next step will be encrypted and
-                password-protected.
+                that will be created with your permission in the next step will be
```

Comment:
how will this read if dataExportEncrypted==false?

Isn't it:
> The export that will be created with your permission in the next step will be.

Should  we hide the entire sentence?

Reply:
good catch, updated so it hides the complete sentence